server/src/main/java/org/apache/cloudstack/ha/HAManagerImpl.java

```diff
@@ -298,14 +308,10 @@ public Boolean isVMAliveOnHost(final Host host) throws Investigator.UnknownVM {
         final HAConfig haConfig = haConfigDao.findHAResource(host.getId(), HAResource.ResourceType.Host);
         if (haConfig != null) {
             if (haConfig.getState() == HAConfig.HAState.Fenced) {
-                if (LOG.isDebugEnabled()){
-                    LOG.debug("HA: Host is fenced " + host.getId());
-                }
+                LOG.debug(String.format("HA: Host [%s] is fenced.", host.getId()));
```

Comment:
please put those checks back in

Reply:
I was wondering if some of the HA messages should be logged as DEBUG or INFO levels. I think that there are some quite important information to raise for ADMINs regardless of they wanting a debug, info, or trace log level. 

For example, when a host gets fenced.

What do you think @DaanHoogland @GutoVeronezi? Does that make sense?

Reply:
I totally agree that debug level has been highly abused in the project. lot's of production environments use debug level by default for that reason. This should not have an impact on this PR though.